lib/logstash/filters/mutate.rb

```diff
@@ -228,6 +240,13 @@ def remove(event)
   end # def remove
 
   private
+  def keep(event)
+    event.to_hash.each do |name, value|
```

Comment:
we might want to allow nested field references using the accessors notation here. looping through the high level 
names would make us miss these deeper level fields